src/main/java/com/cst438/controller/EnrollmentController.java

```diff
@@ -6,37 +6,56 @@
 import org.springframework.http.HttpStatus;
 import org.springframework.web.bind.annotation.*;
 import org.springframework.web.server.ResponseStatusException;
-
+import com.cst438.domain.EnrollmentRepository;
+import com.cst438.domain.Enrollment;
 import java.util.ArrayList;
 import java.util.List;
 
 @RestController
 @CrossOrigin(origins = "http://localhost:3000")
 public class EnrollmentController {
+    @Autowired
+    private EnrollmentRepository enrollmentRepository;
 
     // instructor downloads student enrollments for a section, ordered by student name
     // user must be instructor for the section
     @GetMapping("/sections/{sectionNo}/enrollments")
     public List<EnrollmentDTO> getEnrollments(
             @PathVariable("sectionNo") int sectionNo ) {
 
-        // TODO
-		//  hint: use enrollment repository findEnrollmentsBySectionNoOrderByStudentName method
-        //  remove the following line when done
-        return null;
+        List<Enrollment> enrollments = enrollmentRepository.findEnrollmentsBySectionNoOrderByStudentName(sectionNo);
+        List<EnrollmentDTO> enrollmentDTOs = new ArrayList<>();
+        for (Enrollment enrollment : enrollments) {
+            enrollmentDTOs.add(new EnrollmentDTO(
+                    enrollment.getEnrollmentId(),
+                    enrollment.getGrade(),
+                    enrollment.getUser().getId(),
+                    enrollment.getUser().getName(),
+                    enrollment.getUser().getEmail(),
+                    enrollment.getSection().getCourse().getCourseId(),
+                    enrollment.getSection().getCourse().getTitle(),
+                    enrollment.getSection().getSecId(),
+                    enrollment.getSection().getSectionNo(),
+                    enrollment.getSection().getBuilding(),
+                    enrollment.getSection().getRoom(),
+                    enrollment.getSection().getTimes(),
+                    enrollment.getSection().getCourse().getCredits(),
+                    enrollment.getSection().getTerm().getYear(),
+                    enrollment.getSection().getTerm().getSemester()
+            ));
+        }
+        return enrollmentDTOs;
     }
 
     // instructor uploads enrollments with the final grades for the section
     // user must be instructor for the section
     @PutMapping("/enrollments")
     public void updateEnrollmentGrade(@RequestBody List<EnrollmentDTO> dlist) {
```

Comment:
Pretty sure we are supposed to pass in the instructor email for now to this method. See point #9 in the instructions.

Reply:
Furthermore, I think we're supposed to be validating that the instructor email belongs to the instructor in charge of the section. See the comment above the method.

Reply:
I'm not sure whether we actually need to do anything here, though...

Reply:
Let's ignore this, I guess.